types/index.d.ts

```diff
@@ -86,3 +86,29 @@ declare module '@react-native-community/async-storage' {
 
   export default AsyncStorage;
 }
+
+declare module '@react-native-community/async-storage/jest/async-storage-mock' {
+  type KeysType = Array<string>;
+  type KeyValueType = Array<Array<string>>;
+  type CallbackType = ((error?: Error) => void) | void;
+  type ItemGetCallbackType = ((error?: Error, def?: string) => void) | void;
+  type ResultCallbackType = ((error?: Error, keyValueType?: KeyValueType) => void) | void;
+
+  export interface AsyncStorageMockStatic {
```

Comment:
Duplicating `AsyncStorageStatic` seems like the wrong thing to do. Can we just reuse `AsyncStorageStatic`?

Reply:
cc @januswel